app/code/community/Pulchritudinous/Queue/etc/config.xml

```diff
@@ -32,7 +32,7 @@
 <config>
     <modules>
         <Pulchritudinous_Queue>
-            <version>1.0.1</version>
+            <version>1.0.4</version>
```

Comment:
Should change major i think. These are breaking changes. ```2.0.0```?

Reply:
Git tag will be switched to 3.0 to avoid people from accidentally updating to these breaking changes. The module version is unnecessary to switch to a higher version due to you can't limit the update process in Magento.